lib/oli_web/pow/author_context.ex

```diff
@@ -27,6 +28,43 @@ defmodule OliWeb.Pow.AuthorContext do
     |> maybe_delete_author_cached()
   end
 
+  @doc """
+  Overrides the default Pow.Ecto.Context `create`.
+  """
+  @impl true
+  def create(params) do
+    case Accounts.get_author_by_email(params["email"]) do
+      %Author{email: email} = author ->
+        if author.email_confirmed_at,
+          do:
+            Oli.Email.create_email(
+              email,
+              "Account already registered",
+              "email_already_registered.html",
+              %{
+                url: Routes.authoring_pow_session_path(OliWeb.Endpoint, :new),
+                forgot_password:
+                  Routes.authoring_pow_reset_password_reset_password_path(OliWeb.Endpoint, :new)
+              }
+            )
+            |> Oli.Mailer.deliver_now()
```

Comment:
NIT: We have the `Oli.Email` module that contains email-related functionalities. Perhaps relocating the `create_email` function from its current location to the mentioned module, and renaming it to something like `email_has_been_taken`. What do you think?